demo/src/app/app.component.ts

```diff
@@ -1,10 +1,17 @@
-import {Component} from '@angular/core';
+import {Component, OnInit} from '@angular/core';
+import {Analytics} from './shared/analytics/analytics';
 
 import '../style/app.scss';
 
 @Component({
   selector: 'ngbd-app',
   templateUrl: './app.component.html'
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
+  constructor(private _analytics: Analytics) {
+  }
+
+  ngOnInit(): void {
+    this._analytics.trackPageViews();
```

Comment:
Why do we need it? I would expect that simply including the `Analytics` as a provider should do the trick.